aws_cloudwatch.go

```diff
@@ -747,9 +791,12 @@ func migrateCloudwatchToPrometheus(cwd []*cloudwatchData) []*PrometheusMetric {
 				includeTimestamp = false
 			}
 			if exportedDatapoint != nil {
+
+                promLabels := createPrometheusLabels(c)
+                recordLabelsForMetric(name, promLabels)
```

Comment:
`go fmt` would probably resolve differences in indentation.

Reply:
> We have a bit same kind of situation in
> 
> https://github.com/ivx/yet-another-cloudwatch-exporter/blob/c174df726bea3f65aa9319cd15b03703d5216f5f/aws_cloudwatch.go#L651
> 
> where we add bunch of labels to all apigateway metrics to avoid these problems, so this is not limited to `aws_ecs_*`

ok we should be able to remove that block then since this should handle that case.

Reply:
much appreciated!